themes/light/nes.css

```diff
@@ -0,0 +1,139 @@
+  --article-color-code-foreground: #ad5100;
+  --article-color-code-background: #ffa14d21;
+
+  /* Full code blocks (formatted segments) */
+  --article-pre-color: white;
+  --article-pre-border: #44ff6d;
+  --article-pre-border-thickness: 2px;
+  --article-pre-background: #06150a;
+  /* Formatted code keywords! */
+  --hljs-keyword: #42a3ff;
+  --hljs-string: #ffff65;
+  --hljs-comment: #44ff6d9e;
+  --hljs-title: white;
+  --hljs-params: white;
+  --hljs-number: white;
+  --hljs-built_in: white;
+  --hljs-literal: white;
+
+  /* Onto the toolbar, at the bottom of the menu */
+  /* Color for the icons when not selected */
+  --toolbar-color: #000000b4;
+  --toolbar-hover-color: black;
+
+  /* Highlight color for each section (used as background for it and for the icon) */
+  --toolbar-lang-color: #ff624a;
+  --toolbar-theme-color: #16ce3e;
+  --toolbar-sync-color: #eccb0e;
+  --toolbar-info-color: #445dff;
+
+  /* Color on top of the particular toolbar color defined above (be sure to check accessability!) */
+  --language-color: #610d00;
+  --theme-color: #064c16;
+  --sync-info-color: #302903;
+  --info-color: white;
+}
+
+/* Define additional css rules below! The body ones are just a quick start for changing colors and some common properties. Do whatever you'd like! */
+
+select.theme-selector {
+  /* width: 280px; */
+}
+
+a,
+div.directory-name,
+.toolbar .toolbarIcon {
+  cursor: url(data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAACAAAAAgCAYAAABzenr0AAAAzElEQVRYR+2X0Q6AIAhF5f8/2jYXZkwEjNSVvVUjDpcrGgT7FUkI2D9xRfQETwNIiWO85wfINfQUEyxBG2ArsLwC0jioGt5zFcwF4OYDPi/mBYKm4t0U8ATgRm3ThFoAqkhNgWkA0jJLvaOVSs7j3qMnSgXWBMiWPXe94QqMBMBc1VZIvaTu5u5pQewq0EqNZvIEMCmxAawK0DNkay9QmfFNAJUXfgGgUkLaE7j/h8fnASkxHTz0DGIBMCnBeeM7AArpUd3mz2x3C7wADglA8BcWMZhZAAAAAElFTkSuQmCC)
+      14 0,
+    pointer;
+}
+
+input {
+  cursor: inherit;
+}
+h1,
+h2,
+h3 {
+  font-weight: bold !important;
+}
+
+select.theme-selector {
+  position: relative;
+  width: calc(100% - 8px);
+  margin: 4px;
+  border-image-slice: 2;
+  border-image-width: 2;
+  border-image-repeat: stretch;
+  border-image-source: url('data:image/svg+xml;utf8,<?xml version="1.0" encoding="UTF-8" ?><svg version="1.1" width="5" height="5" xmlns="http://www.w3.org/2000/svg"><path d="M2 1 h1 v1 h-1 z M1 2 h1 v1 h-1 z M3 2 h1 v1 h-1 z M2 3 h1 v1 h-1 z" fill="rgb(33,37,41)" /></svg>');
+  border-image-outset: 2;
+  /* width: 100%; */
+  padding: 0.5rem 2.5rem 0.5rem 1rem;
+  cursor: url(data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAACAAAAAgCAYAAABzenr0AAAAzElEQVRYR+2X0Q6AIAhF5f8/2jYXZkwEjNSVvVUjDpcrGgT7FUkI2D9xRfQETwNIiWO85wfINfQUEyxBG2ArsLwC0jioGt5zFcwF4OYDPi/mBYKm4t0U8ATgRm3ThFoAqkhNgWkA0jJLvaOVSs7j3qMnSgXWBMiWPXe94QqMBMBc1VZIvaTu5u5pQewq0EqNZvIEMCmxAawK0DNkay9QmfFNAJUXfgGgUkLaE7j/h8fnASkxHTz0DGIBMCnBeeM7AArpUd3mz2x3C7wADglA8BcWMZhZAAAAAElFTkSuQmCC)
+      14 0,
+    pointer;
+  border-radius: 0;
+  outline-color: #e7e7e7;
+  -webkit-appearance: none;
+  -moz-appearance: none;
+  appearance: none;
```

Comment:
Themes must render the theme selector using the appropriate web appearance, in order to ensure compatibility with expected UX on mobile browsers.
```suggestion
```